docs/plugins/snacks/terminal.md

```diff
@@ -0,0 +1,80 @@
+# Terminal Integration
+
+Enhanced terminal functionality within Neovim with better integration and management.
+
+## Features
+
+- **Floating Terminals**: Beautiful floating terminal windows
+- **Terminal Management**: Easy creation and switching between terminals
+- **Shell Integration**: Better shell command execution
+- **Window Controls**: Resize, move, and manage terminal windows
+- **Session Persistence**: Maintain terminal sessions
+
+## Configuration
+
+```lua
+terminal = { 
+  enabled = true,
+  shell = nil,           -- Use default shell
+  position = "float",    -- Terminal position
+  size = { width = 0.8, height = 0.8 },
+}
+```
+
+## Usage
+
+### Commands
+- `:Terminal` - Open floating terminal
+- `:TerminalSplit` - Open terminal in horizontal split
+- `:TerminalVsplit` - Open terminal in vertical split
+- `:TerminalTab` - Open terminal in new tab
```

Comment:
> - `:Terminal` - Open floating terminal
> - `:TerminalSplit` - Open terminal in horizontal split
> - `:TerminalVsplit` - Open terminal in vertical split
> - `:TerminalTab` - Open terminal in new tab

These commands don't appear to exist, the only similar thing that exists is the default `:terminal` command